deployment/live/gcp/cloudbuild/terragrunt.hcl

```diff
@@ -0,0 +1,22 @@
+locals {
+  env        = path_relative_to_include()
+  project_id = get_env("GOOGLE_PROJECT", "transparency-dev")
+  location   = get_env("GOOGLE_REGION", "us-central1")
```

Comment:
very nit, region to align with tessera repo

Reply:
Do you mean the `location` cannot be changed?

Reply:
I see. They seem to be interchangeable. Most terraform resources use `location` as the arguments.